### DISCUSSION TITLE
Green Power Support

### DISCUSSION THREAD

**Discussion by tfriedlx** [Koenkk/zigbee2mqtt#7650] [Devices]
I'm currently using a CC2531 stick and my Green Power device works only with having a Philips Hue Bulb available. 

Would switching to a different Zigbee stick allow me to use Green Power devices without Hue bulbs?

I've contacted Electrolama - the makers of the zzh stick and they wrote me this:
_I think there's some work being done on support Green Power devices but this is very much a firmware/software question so I suggest raising this on the relevant support forum/issue tracker for the software you are planning on using. Hardware-wise, CC2652R (chip used on zzh) should be able to support this._

So would using the zzh stick allow me to connect to Green Power devices without a Hue bulb?

Thanks!

**Comment by Koenkk** [comment]
> So would using the zzh stick allow me to connect to Green Power devices without a Hue bulb?

No, you will still need a green power router (e.g. a Hue bulb)

**Reply by tfriedlx** [reply]
Thanks for the info!

Do you maybe know if there is a green power router thats not a light bulb? Anything less bulky I could put on a shelf or something like that?

**Reply by Koenkk** [reply]
I believe hue power plugs are also green power routers.

**Reply by sjorge** [reply]
Ubisys S1 and S2 are also green power routers, those go behind a wall switch but they are rather expensive sadly :(
